modules/appeals_api/spec/workers/add_icn_updater_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
```

Comment:
probably should add some tests for different failure paths as well (i.e., mainly if address attributes are missing)

Reply:
Did that (along with a small logic fix exposed by the test) [here](https://github.com/department-of-veterans-affairs/vets-api/pull/10574/commits/71ba89e36d1ea63e8cad090674b00877420296be)